### ISSUE TITLE
Can't compile successfully

### ISSUE THREAD

**Issue by zhangshuaiyaoyao** [yhtang/MESO#1]
Through Kirill Lykov's introduction, I came into contact with the USER-MESO2.0 written by you. But after several days of trying, I still failed to compile. Therefore, I want to ask here whether MPICH, CUDA, etc. are used in the software, and whether these have specific version requirements. Could you please explain in detail the installation and compilation process of this package? In addition, I have been using lammps before. Your package is written on the basis of lammps, so the difference between the compilation of its .in file and lammps What are there?

The following are two problems that occurred during the installation and compilation process:
1.error while loading shared libraries: libcudart.so.8.0: cannot open shared object file: can not open
2.lmp_meso: error while loading shared libraries: libmpi_cxx.so.1: cannot open shared object file: No such file or directory
When I first started using CUDA9.0, there was problem 1, but after switching to CUDA8.0, there was no problem 1, but problem 2 occurred.

**Comment by KirillLykov** [comment]
Could you check where are these libraries on you machine? Depending on the system, it might be different folders. One can try to do `find . -name "libcudart.so.*" for example to find one of them. After that, add this path to `LD_LIBRARY_PATH`.

**Comment by zhangshuaiyaoyao** [comment]
Thank you very much for your reply. After several days of trying, I successfully compiled it.
Here I will introduce the problems and solutions I encountered in detail:
1.error while loading shared libraries: libcudart.so.8.0: cannot open shared object file:
I installed cuda9.0 at the beginning, and there is no libcudart.so.8.0 file in it. I installed cuda8.0 instead of reporting this error, but problem 2 occurred.
2.lmp_meso: error while loading shared libraries: libmpi_cxx.so.1: cannot open shared object file: No such file or directory
At first I thought it was a problem with the MPICH version. I tried multiple versions such as 3.3.2, 1.4, 1.4.1, etc., but still did not solve the problem. Finally, search for libmpi_cxx.so.1 on the http://pkgs.org website, and successfully compile after installing libopenmpi1.10.
Finally, thank you again for your reply.
